sdk/tests/conformance2/rendering/blitframebuffer-multisampled-readbuffer.html

```diff
@@ -140,27 +140,43 @@
     gl.enableVertexAttribArray(positionLoc);
     // Draw two triangles. They are almost right-triangles: one is actually an obtuse triangle. The other is actually an acute triangle.
     // This means that some egdes of the triangles almost parallel with x-axis and y-axis. So they can show the alias.
-    var positions = new Float32Array([
+    /* var positions = new Float32Array([
          0.85, -0.90,
         -0.90,  0.85,
         -0.87, -0.87,
          0.90, -0.85,
          0.93,  0.93,
         -0.85,  0.90,
+    ]);*/
+    var positions = new Float32Array([
+         1.0, -1.0,
+        -1.0,  1.0,
+        -1.0, -1.0,
+         1.0, -1.0,
+         1.0,  1.0,
+        -1.0,  1.0,
     ]);
     var positionBuffer = gl.createBuffer();
     gl.bindBuffer(gl.ARRAY_BUFFER, positionBuffer);
     gl.bufferData(gl.ARRAY_BUFFER, positions, gl.STATIC_DRAW);
     gl.vertexAttribPointer(positionLoc, 2, gl.FLOAT, false, 0, 0);
 
     gl.enableVertexAttribArray(colorLoc);
-    var colors = [
+    /* var colors = [
         [255, 0, 0, 255],
         [0, 255, 0, 255],
         [0, 0, 255, 255],
         [253, 122, 5, 255],
         [6, 252, 120, 255],
         [118, 4, 254, 255]
+    ];*/
+    var colors = [
+        [255, 0, 0, 255],
+        [0, 255, 0, 255],
+        [0, 0, 255, 255],
+        [255, 0, 0, 255],
+        [0, 0, 255, 255],
+        [0, 255, 0, 255],
```

Comment:
Let's keep the test simple and draw a rectangle, so as to avoid antialiasing issues as much as possible.

Something seems wrong with this test. If I change all of the per-vertex colors to (200, 0, 0) -- i.e., solid red, which I assume should fall within the sRGB color gamut (not sure whether the fully-saturated colors do) -- then the test fails badly, and the colors are visibly different on my MacBook Pro with NVIDIA GPU, on a Chromium continuous build containing your fix https://codereview.chromium.org/2286593002/ .

Can you double-check the logic throughout? With the above change, does the test fail on your Linux workstation with Intel GPU?

I assume that if the conversions between linear and sRGB are all correct in this test, that it shouldn't even need a tolerance of 7 or 8.